### ISSUE TITLE
Support 'Yields' in Google-style docstrings

### ISSUE THREAD

**Issue by jvfe** [mkdocstrings/pytkdocs#89]
I was using mkdocstrings, but I think this issue is specifically related to pytkdocs. 

**Is your feature request related to a problem? Please describe.**

Yes. I had a docstring of a generator that I was using mkdocstrings (v0.14.0) to compile the documentation for, something like:
```py
    """
    Get the next number in the sequence

    Args:
        n (int): The upper limit of the range to generate, from 0 to `n` - 1.

    Yields:
        int: The next number in the range of 0 to `n` - 1.
    """
```
[Like specified in the Google style-guide](https://google.github.io/styleguide/pyguide.html#doc-function-returns).

But it was being rendered like `!!! yields !!! int ...`

**Describe the solution you'd like**

I'd like to be able to use `Yields` or `Yield` for generators.

**Describe alternatives you've considered**

Replacing `Yields` for `Returns` works as expected, but that is not an optimal solution. 

**Additional context**

I've noticed [napoleon](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#docstring-sections) seems to support both `Yields` and `Yield` for these cases.

And thanks for making mkdocstrings, it's really saved me a lot of time.

**Comment by pawamoy** [comment]
Hello @jvfe, thank you for the detailed feature request! I'm totally in favor of adding support for Yields section :slightly_smiling_face:

**Comment by jvfe** [comment]
Sure, I could try giving it a go! 

What did you have in mind? Would it be implementing a `Section.Type.YIELD` and use `self.read_return_section` here ↓?
https://github.com/pawamoy/pytkdocs/blob/a298616c2db23833b4877c67725cbbfde5a43c78/src/pytkdocs/parsers/docstrings/google.py#L7-L22
https://github.com/pawamoy/pytkdocs/blob/a298616c2db23833b4877c67725cbbfde5a43c78/src/pytkdocs/parsers/docstrings/google.py#L40-L46

**Comment by pawamoy** [comment]
Oh, I realize I never answered, so sorry :bow: 
Yes it should mimic other existing sections. I'll take care of it right now.

**Comment by pawamoy** [comment]
It will be released in version 0.13 as this would not be caught by mkdocstrings 0.16.
